core/src/main/resources/schema/sqlserver/sqlserver-create-schema.sql

```diff
@@ -40,3 +40,24 @@ CREATE TABLE "snapshot" (
     PRIMARY KEY ("persistence_id", "sequence_number")
   )
 
+-- Create Sequence Object
+CREATE SEQUENCE global_offset
+    START WITH 1
+    INCREMENT BY 1;
+
+CREATE TABLE durable_state
+(
+    "global_offset"         BIGINT
+        CONSTRAINT [df_global_offset] DEFAULT
+        (NEXT VALUE FOR global_offset),
+    "persistence_id"        VARCHAR(255)   NOT NULL,
+    "revision"              NUMERIC(10, 0) NOT NULL,
+    "state_payload"         VARBINARY(MAX) NOT NULL,
+    "state_serial_id"       INTEGER        NOT NULL,
+    "state_serial_manifest" VARCHAR(MAX),
```

Comment:
can we use text, varchar(max) is strange

Reply:
and pg use VARCHAR(255),it should be this?

Reply:
Same as `event_journal `, the choice of a new column needs another separate PR and more investigation.